index.js

```diff
@@ -26,6 +26,10 @@ function attachComments(text, ast, opts) {
 function format(text, opts) {
   const ast = parse(text, opts);
   attachComments(text, ast, opts);
+  return formatAST(ast, opts);
+}
+
+function formatAST(ast, opts) {
```

Comment:
Do you mind not making a formatAST function here and duplicating the code in debug instead? This way we can build all the variations using the primitives and not have weird mixes of functions that are split in the middle.